test/groovy/util/JenkinsResetDefaultCacheRule.groovy

```diff
@@ -27,7 +27,23 @@ class JenkinsResetDefaultCacheRule implements TestRule {
             @Override
             void evaluate() throws Throwable {
                 DefaultValueCache.reset()
+
+                MetaMethod oldCreateInstanceFromPersistence = DefaultValueCache.metaClass.
+                    getStaticMetaMethod("createInstanceFromPersistence", Script)
+                MetaMethod oldPersistDefaults = DefaultValueCache.metaClass.
+                    getStaticMetaMethod("persistDefaults", [Script, Map, List])
+
+                DefaultValueCache.metaClass.static.createInstanceFromPersistence = { Script s ->
+                    return null
+                }
+                DefaultValueCache.metaClass.static.persistDefaults = { Script s, Map dv, List cd ->
+                    return null
+                }
+
                 base.evaluate()
+
+                DefaultValueCache.metaClass.static.createInstanceFromPersistence = oldCreateInstanceFromPersistence
```

Comment:
Also here we restore without taking the signature into account, but when remembering the original methods we take the signature into account. Is that OK?